pkg/kubernetes/templating.go

```diff
@@ -9,7 +9,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/validation"
 )
 
-const defaultNamespace = "istio-system"
+// DefaultNamespace to generate configuration for
+const DefaultNamespace = "istio-system"
```

Comment:
remember to squash/merge this commit (and the one changing the tests) with the commits that introduce these changes